released/packages/coq-infotheo/coq-infotheo.0.0.5/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "reynald.affeldt@aist.go.jp"
+homepage: "https://github.com/affeldt-aist/infotheo"
+bug-reports: "https://github.com/affeldt-aist/infotheo/issues"
+dev-repo: "git+https://github.com/affeldt-aist/infotheo.git"
+license: "GPLv3"
+authors: [
+  "Reynald Affeldt"
+  "Manabu Hagiwara"
+  "Jonas Senizergues"
+  "Jacques Garrigue"
+  "Kazuhiko Sakaguchi"
+  "Taku Asai"
+  "Takafumi Saikawa"
+  "Naruomi Obata"
+  "Erik Martin-Dorel"
+  "Ryosuke Obi"
+  "Mitsuharu Yamamoto"
+]
+build: [
+  ["coq_makefile" "-f" "_CoqProject" "-o" "Makefile"]
+  [make]
+  [make "-C" "extraction" "tests"] {with-test}
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" {>= "8.10" & < "8.11~"}
+  "coq-mathcomp-field" {>= "1.9.0" & < "1.10~"}
+  "coq-mathcomp-analysis"   {>= "0.2.0" & <= "0.2.2"}
+]
+synopsis: "Infotheo"
+description: """
+a Coq formalization of information theory and linear error-correcting codes
+"""
+tags: [
+  "category:Computer Science/Data Types and Data Structures"
+  "keyword: information theory"
+  "keyword: probability"
+  "keyword: error-correcting codes"
+  "logpath:infotheo"
+  "date:2019-11-09"
+]
+url {
+  http: "https://github.com/affeldt-aist/infotheo/archive/0.0.5.tar.gz"
+  checksum: "43b937f49aafaae9d537e61c77345f8f"
```

Comment:
```suggestion
  checksum: "sha512=0ba8876041b815a624b49721521d31334858c6ecce941b5ba8ba29e374bd9bb300be9dc197dba0478a0def20154c3363e46bfe795dfff736a20d31dd05a8ef9a"
```
We are trying to deprecate MD5 for checksums.

Reply:
done